src/lib.rs

```diff
@@ -3,122 +3,69 @@ use pyo3::prelude::{pymodule, wrap_pyfunction, PyModule, PyResult, Python};
 mod addrs;
 mod api_crypto;
 mod binding_utils;
-mod certif;
 mod data;
 mod enumerate;
 mod file_operations;
 mod ids;
-mod invite;
 mod local_device;
-mod local_manifest;
-mod manifest;
-mod message;
 mod protocol;
 mod regex;
 mod runtime;
 mod time;
 mod trustchain;
 
-/// A Python module implemented in Rust.
-#[pymodule]
-#[pyo3(name = "_parsec")]
-fn entrypoint(py: Python, m: &PyModule) -> PyResult<()> {
-    m.add_class::<addrs::BackendAddr>()?;
-    m.add_class::<addrs::BackendOrganizationAddr>()?;
-    m.add_class::<addrs::BackendActionAddr>()?;
-    m.add_class::<addrs::BackendOrganizationBootstrapAddr>()?;
-    m.add_class::<addrs::BackendOrganizationFileLinkAddr>()?;
-    m.add_class::<addrs::BackendInvitationAddr>()?;
-    m.add_class::<addrs::BackendPkiEnrollmentAddr>()?;
-    m.add_function(wrap_pyfunction!(addrs::export_root_verify_key, m)?)?;
+fn add_data_module(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
```

Comment:
Could you move this function into `data/mod.rs` ?

Reply:
Sure